tests/unit/utils/includeSpecs.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
```

Comment:
question: Where is this used?

Reply:
HI @jimee02 , initially I want include specs files manually, after choose other option, and left here as function if could be required in the future, but will remove it for now.